### ISSUE TITLE
from_mp3 stops execution when script is run in the background.

### ISSUE THREAD

**Issue by egreenfield** [jiaaro/pydub#243]
### Steps to reproduce
1. put the following simple code into a file called transform.py
transform.py:
```
import pydub
def transform(sourceFile,destFile):

    print("loading")
    song = pydub.AudioSegment.from_mp3(sourceFile)
    print("amping")
    song = song + 5
    print("speeding")
    song = pydub.effects.speedup(song,1.2)
    print("writing")
    song.export(destFile, format="mp3")
 
def testTransform():
    transform("../output/raw.mp3","../output/testTransform.mp3")

if __name__ == "__main__":
    testTransform()
```    
2. run the file from a bash shell, in the background:
`python3 transform.py &
`
### Expected behavior
script should run, all printed output should show, testTransform.mp3 should be generated.

### Actual behavior
only the 'loading' output shows. 
shell reports the script as stopped (presumably something in the from_mp3 call forced the script to suspend in some way)
if you 'fg' the script in the shell, it will complete as expected.

### Your System configuration
- Python version: 3.6.4
- Pydub version: 0.20.0
- ffmpeg or avlib?: ffmpeg
- ffmpeg/avlib version: 3.4.1

tested on both osx and raspberry pi (bash 3.2.57 on osx and 4.4.12 on rpi)

### Is there an audio file you can include to help us reproduce?
works with any mp3 file I've tried.

thanks!

**Comment by jiaaro** [comment]
Very odd - I'm not sure why that would be the case. Loading the mp3 entails creating a named temporary file, and making a subprocess call, both of which _should_ (as far as I know) be allowed.

Did you ever learn any more about this issue?

**Comment by egreenfield** [comment]
Nope.  I wrote a quick test of similar functionality using an ffmpeg
wrapper from a nodejs process, and it had no problems (didn't exhibit the
same behavior).  Since my project was relatively small, I rewrote it in
node, and moved on.

Thanks.

**Comment by aramcpp** [comment]
I think I may have found the reason for this issue. I faced the same issue and here what I found out

I enabled logging and got this log when loading from_file
`subprocess.call(['ffmpeg', '-y', '-f', 'mp3', '-i', 'some_file.mp3', '-acodec', 'pcm_s16le', '-vn', '-f', 'wav', '-'])`
And I tried to run this
And when running in background process hangs.
But if we add `-nostdin` everything works.

Then I took a little snippet from you source code and created some script looking like this.
```

import subprocess

p = subprocess.Popen(['ffmpeg',  '-y', '-f', 'mp3', '-i', 'some_file.mp3', '-acodec', 'pcm_s16le', '-vn', '-f', 'wav', '-'],
                     stdin=None,
                     stdout=subprocess.PIPE,
                     stderr=subprocess.PIPE)
p_out, p_err = p.communicate(input=None)

if p.returncode != 0 or len(p_out) == 0:
    if close_file:
        file.close()
    raise CouldntDecodeError(
        "Decoding failed. ffmpeg returned error code: {0}\n\nOutput from ffmpeg/avlib:\n\n{1}".format(
            p.returncode, p_err.decode(errors='ignore') ))

p_out = bytearray(p_out)

print(p_out)

print('done')
```

Same behavior as with the last example. And yes adding `-nostdin` fixes it.

To conclude. The problem is with ffmpeg hanging not the pydub.

On pydub side can be "fixed" like this.

`AudioSegment.from_mp3(file_path, parameters=['-nostdin'])`

Hope this is helpful.
